docs/release-notes/2.24.0.md

```diff
@@ -19,6 +19,19 @@
 </table>
 
 Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+## Release Highlights
+- **Better PostgreSQL monitoring**
+  - Starting from this version of PMM, it will support monitoring of PostgreSQL 14. Both versions - Community and [Percona Distribution for PostgreSQL](https://www.percona.com/software/postgresql-distribution).
+  - This version of PMM has also a significant improvement in exposed data and additional features if you monitor PostgreSQL together with the [pg_stat_monitor](https://github.com/percona/pg_stat_monitor) plugin (Part of Percona distribution for PostgreSQL). We added such features:
```

Comment:
```suggestion
  - This version of PMM has also a significant improvement in exposed data and additional features if you monitor PostgreSQL together with the [pg_stat_monitor](https://github.com/percona/pg_stat_monitor) extension (Part of Percona distribution for PostgreSQL). We added such features:
```

Reply:
it either extension or tool

Reply:
🤦‍♂️ 
EXTENSION... is how this is named. Thanks for spotting this